lua/catppuccin/init.lua

```diff
@@ -40,6 +40,7 @@ local M = {
 		integrations = {
 			alpha = true,
 			cmp = true,
+			copilot_vim = true,
```

Comment:
Just one more change sorry, I don't believe we're adding more integrations defaulting to `true` anymore. I'd appreciate if you could remove this line.

Reply:
No problem, totally understand.